picard/ui/sortablecheckboxlist.py

```diff
@@ -101,6 +106,33 @@ def _emit_changed(self):
         if not self.__no_emit:
             self.changed.emit(self.__items)
 
+    def set_item_marking(self, row, value):
```

Comment:
It's not really required for this PR as I can just remove all items.. I implemented it initially and didn't want to remove the feature of delayed delete. Should I?